src/core/Rational.pm6

```diff
@@ -298,8 +298,81 @@ my role Rational[::NuT = Int, ::DeT = ::("NuT")] does Real {
             ?? $!numerator
             !! self;
     }
-
-    multi method round(::?CLASS:D: --> Int:D) {
+    multi method round(::?CLASS:D: :$HALF-DOWN! --> Int:D) {
```

Comment:
Required truth-only booleans need to be declared `Bool:D :$name! where .so`, not just `:$name!`

Reply:
Corrected in b4d3723